dependencies/dependencies.go

```diff
@@ -50,12 +53,30 @@ func (d Dependencies) URLValidator() (url.Validator, error) {
 	return nil, errors.New(codes.Unimplemented, "url validator uninitialized in dependencies")
 }
 
+// newDefaultTransport creates a new transport with sane defaults.
+func newDefaultTransport() *http.Transport {
+	// These defaults are copied from http.DefaultTransport.
+	return &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
+		DialContext: (&net.Dialer{
+			Timeout:   30 * time.Second,
+			KeepAlive: 30 * time.Second,
+			// DualStack is deprecated
+		}).DialContext,
+		MaxIdleConns:          100,
+		IdleConnTimeout:       90 * time.Second,
+		TLSHandshakeTimeout:   10 * time.Second,
+		ExpectContinueTimeout: 1 * time.Second,
+		MaxIdleConnsPerHost:   runtime.GOMAXPROCS(0) + 1,
```

Comment:
What is the reasoning for this default? You had it set to `100` earlier.

Reply:
I pulled this from a commit on another branch, but I just checked the defaults in a doc elsewhere and they changed to 100. I'll update that now.